test/commands/new.test.js

```diff
@@ -62,4 +62,33 @@ describe("cli: gluestick new", function () {
     }
   });
 
+  it("should generate a test for all of the initial components and containers", () => {
+    const fakeNpm = sinon.stub(npmDependencies, "install");
```

Comment:
why not have this and the restore in the `beforeEach` and `afterEach` you already have?